### DISCUSSION TITLE
Nx Project Setup / Concept

### DISCUSSION THREAD

**Discussion by APOSWE** [bennymeg/nx-electron#248] [Q&A]
Hey!
I've some conceptual questions. I've been trying to find some information and help on the web for hours now, unfortunately without success. Hopefully you can help me! 🙏
I setup a nx15 workspace with nx-electron and I was able to start the native program. (as well I setup an angular project with electron)
That electron is working I had to start my test-app and paste the url of the test-app in the constants.ts of the electron-app.

How is it possible to bundle this two apps together? As I've seen in the "normal"-angular project, electron was kind of wrapping around the app. 

Is there an example available that I could use as a guide?

Many thanks in advance!
Alex

**Comment by bennymeg** [comment]
If I understood your question correctly, you have a v15 nx-electron project with angular application as a frontend project.
You are able to serve the application correctly, then, it seems all ou have to do is:
1. build your angular frontend application
2. build your electron backend application
3. package them together with nx-electron (I recommand installing nx-console for vs-code, it will provide GUI for all the available commands).

**Reply by APOSWE** [reply]
exactly! Normally I wanted to setup a nx v16 monorepo but for the compatibility with nx-electron I built an v15 proj to test electron...
therefore I was just using the example-template of the nx-app. so i can build both error-free.

Point 3. is i guess the main topic. the packaging is not working.. this time I used the nx-console extension.. that's easier 🙂
![image](https://github.com/bennymeg/nx-electron/assets/115154268/17625990-aa1b-4c94-9aa6-49f893ae6af7)
![image](https://github.com/bennymeg/nx-electron/assets/115154268/47019201-c93b-4d91-a204-bc89f2bfbd65)
now it's hangin on the rebuilding step 😬
